### DISCUSSION TITLE
Can not hide vidyard player controls

### DISCUSSION THREAD

**Discussion by LoayH98** [cookpete/react-player#1818] [Ideas]
#### Current Behavior
Tried to display a Vidyard video using react-player and the controls can not be hidden

#### Expected Behavior
The native player controls should be disabled and hidden

#### Steps to Reproduce
Try this link for example: https://video.vidyard.com/watch/5p4Dfgys8k7gBwZdCJgJwQ

#### Environment
- [URL attempting to play:](https://video.vidyard.com/watch/5p4Dfgys8k7gBwZdCJgJwQ) 
- Browser: Chrome
- Operating system: Windows

#### Other Information
--

**Comment by amrinder1189** [comment]
This seems to be easy to solve this, let me work on this one.
